src/utils/word.utils.ts

```diff
@@ -48,14 +48,18 @@ const getNumberOfWords = (
   return wordsList.concat().slice(0, numberOfWordsToGet);
 };
 
+// createFillInString creates a word string for the H5P.Blanks content type.
```

Comment:
Use jsdoc when documenting.

```suggestion
/**
 * Creates a word string for the H5P.Blanks content type. 
 * H5P.Blanks expects the input as an HTML string on the format `<from> *<to>*`.
 */
```